starter/script.js

```diff
@@ -1 +1,9 @@
-// put code here!
+confrim ("Hello are you ready to begin this adventure?!");
+
+var age = prompt("What's your age");
+
+if(age > 13) {
+  console.log("Welcome!"");
```

Comment:
The second `"` after welcome is causing a syntax error. That's going to cause the whole script to break